static/js/snap-pages/components/EmptySnapList/EmptySnapList.tsx

```diff
@@ -0,0 +1,69 @@
+import { Col, Link, Row } from "@canonical/react-components";
+
+export const EmptySnapList = () => {
```

Comment:
I think this should be `function EmptySnapList` and export at the bottom of the file so it's consistent with other React components